user_guides/jakartaee/src/main/jbake/content/rules.adoc

```diff
@@ -411,20 +411,23 @@ programmatic behavior of the Runtime to vary from the functional
 programmatic behavior of the Runtime in the absence of that comment.
 
 EE21 Compliance testing for Jakarta EE 10.0 consists of running Jakarta EE 10.0
-TCK and the following Technology Compatibility Kits (TCKs):
-
-* Jakarta Authentication 3.0
-* Jakarta Batch 2.1
-* Jakarta Bean Validation 3.0
-* Jakarta Concurrency 3.0
-* Jakarta Contexts and Dependency Injection 4.0
-* Jakarta Dependency Injection 2.0 
-* Jakarta Faces 4.0
-* Jakarta JSON Processing 2.1
-* Jakarta JSON Binding 3.0
-* Jakarta RESTful Web Services 3.1
-* Jakarta Security 3.0
-* Jakarta XML Binding 4.0 (If XML Binding is supported)
+TCK and the following Technology Compatibility Kits (TCKs). Version details are defined in the Platform EE Specification document (https://jakarta.ee/specifications/platform/10/), see the heading 'Full Jakarta™ EE Product Requirements':
+
+* Jakarta Activation
+* Jakarta Authentication
+* Jakarta Batch
+* Jakarta Bean Validation
+* Jakarta Concurrency
+* Jakarta Contexts and Dependency Injection
+* Jakarta Debugging Support for Other Languages
```

Comment:
Jakarta Activation, Mail, Debugging Support shouldn't be added to this list.  Mail is already tested by Platform TCK which requires Activation to work as well.  Adding Debugging Support for Other Languages seems late to add.

Reply:
These specifications have always been a requirement.
For  Debugging support TCK, I would maintain this was an error in previous documents. 
Prior to EE 10 Activation and Mail were verified compatible via the Platform TCK. The fact these TCKs are now factored out of the Platform TCK necessitates they be listed here.